src/OpenStackNetTests.Live/ContentDeliveryTests.Shared.cs

```diff
@@ -0,0 +1,405 @@
+﻿namespace OpenStackNetTests.Live
```

Comment:
:exclamation: This file is currently only included in the Live tests project. It should be linked to the Unit tests project so a service simulator can be implemented for efficient unit testing of the service functionality.